### ISSUE TITLE
Tic File Processor didn't work on nodelist file

### ISSUE THREAD

**Issue by apamment** [NuSkooler/enigma-bbs#111]
For :bug: bug reports, please fill out the information below plus any additional relevant information. If this is a feature request, feel free to clear the form.

**Short problem description**
Tic processor did not import fsx nodelist, but did import fsx infopack
**Environment**
- [X] I am using Node.js v6.x or higher
- [X] `npm install` reports success
- Actual Node.js version (`node --version`): v6.10.0
- Operating system (`uname -a` on *nix systems): Linux zatanna.andrew.homeunix.org 3.10.0-514.10.2.el7.x86_64 #1 SMP Thu Mar 2 11:21:24 CST 2017 x86_64 x86_64 x86_64 GNU/Linux
- Revision (`git rev-parse --short HEAD`):732e2ee

**Expected behavior**
Should import file

**Actual behavior**
Log file reports success:
```
[2017-03-13T09:01:23.111Z] DEBUG: ENiGMA½ BBS/19029 on zatanna.andrew.homeunix.org: Processing TIC file (tic=/home/andrew/ftn/in_secure/v90r31k2.tic, file=FSXNET.Z76)
[2017-03-13T09:01:23.156Z] DEBUG: ENiGMA½ BBS/19029 on zatanna.andrew.homeunix.org: Cleaned up temporary files (paths=[], sessionId=file_area)
[2017-03-13T09:01:23.233Z] DEBUG: ENiGMA½ BBS/19029 on zatanna.andrew.homeunix.org: TIC imported successfully (tic=/home/andrew/ftn/in_secure/v90r31k2.tic, file=/home/andrew/ftn/in_secure/FSXNET.Z76, area=fsx_node)
```

But it's not in the database. The file is moved to the correct storage area.
**Steps to reproduce**

**Comment by NuSkooler** [comment]
I saw this when I first imported but thought it had been fixed along with another minor change. I'll try to reproduce.

**Comment by NuSkooler** [comment]
Fixed @ f510fca65623850ea87b5dee70111036f3f6b034
